dashboard/common/utils.py

```diff
@@ -103,6 +103,8 @@ def get_myla_globals(request):
 
     if settings.GA_ID:
         google_analytics_id = settings.GA_ID
+    if settings.OT_SCRIPT_DOMAIN:
```

Comment:
With `OT_SCRIPT_DOMAIN=""` in the settings file, I got this log error: 

`student_dashboard        |   File "/code/dashboard/common/utils.py", line 120, in get_myla_globals
student_dashboard        |     "one_trust_script_domain": one_trust_script_domain,
student_dashboard        | UnboundLocalError: local variable 'one_trust_script_domain' referenced before assignment`

Need to initialize one_trust_script_domain variable.

Reply:
Could re-write this as 

`one_trust_script_domain = settings.OT_SCRIPT_DOMAIN if settings.OT_SCRIPT_DOMAIN else ""`

Could _probably_ improve the google_analytics_id to have this pattern too rather than setting it earlier.

Reply:
@jonespm thanks for the code suggestion, applied to both.